fineract-charge/src/main/java/org/apache/fineract/portfolio/charge/domain/Charge.java

```diff
@@ -784,4 +780,11 @@ public int hashCode() {
         return Objects.hash(name, amount, currencyCode, chargeAppliesTo, chargeTimeType, chargeCalculation, chargePaymentMode, feeOnDay,
                 feeInterval, feeOnMonth, penalty, active, deleted, minCap, maxCap, feeFrequency, account, taxGroup);
     }
+
+    public static final String CHARGE_TIME_PARAM_NAME = "chargeTimeType";
+    public static final String CHARGE_CALCULATION_TIME_PARAM_NAME = "chargeCalculationType";
```

Comment:
For `chargeCalculationType` param this should be rather named to `CHARGE_CALCULATION_TYPE_PARAM_NAME`, no?